src/main/java/se/sundsvall/supportmanagement/service/CommunicationService.java

```diff
@@ -291,10 +293,19 @@ public void sendMessageNotification(final ErrandEntity errandEntity, final Sende
 
 		final var request = toMessagingMessageRequest(errandEntity, senderInfo);
 
-		final var message = messagingClient.sendMessage(errandEntity.getMunicipalityId(), request);
+		final var partyId = Optional.ofNullable(request.getMessages())
+			.map(List::getFirst)
+			.map(Message::getParty)
+			.map(MessageParty::getPartyId)
+			.map(UUID::toString)
+			.orElse(null);
+
+		if (Identifier.get() != null && !Identifier.get().getValue().equals(partyId)) {
```

Comment:
Should the uuid-comparison be case insensitive?

(we often use lower case, but I don't think that lower case is a requirement)